openbb_sdk/extensions/stocks/openbb_stocks/fa/fa_router.py

```diff
@@ -191,6 +213,17 @@ def income(
     return CommandOutput(results=Query(**locals()).execute())
 
 
+@router.command(model="IncomeStatementGrowth")
+def income_growth(
```

Comment:
@jmaslek how do you think we should handle this? commands with underscores have perfect sense in python but we tried to avoid them on the terminal. if this command will be exposed to the terminal as is it will break the convention. not sure if urls with underscores on the API are a good thing too.

Reply:
I would prefer the following:
- terminal functions have no underscores
- terminal fnc = sdk fnc

by the coimmutative proprty - sdk has no underscore.  @DidierRLopes your thoughts?